webapp/public/locales/en/data.json

```diff
@@ -30,5 +30,6 @@
     "analyzingmatrix": "Analyzing matrices",
     "copyid": "Copy the matrix id",
     "onMatrixIdCopySuccess": "Matrix id copied !",
-    "jsonFormat": "JSON Format"
+    "jsonFormat": "JSON Format",
+    "graphSelector":  "Seletc a column"
```

Comment:
This is "Columns" (we can select multiple ones).
Also no need to add "select" as this is a select so the user already know that the action is a selection.